src/styles/sidebar/components/group-list-item-out-of-scope.scss

```diff
@@ -0,0 +1,50 @@
+/* The group-list out of scope item. */
+
+.group-list-item-out-of-scope {
+  display: flex;
+  flex-direction: row;
+  flex-grow: 1;
+}
+
+.group-list-item-out-of-scope__item {
+  background-color: $gray-lightest;
+  color: $gray-light;
+}
+
+.group-list-item-out-of-scope__icon--unavailable {
+  fill: $gray-light;
+  float: right;
+  height: 20px;
+  width: auto;
+}
+
+.group-list-item-out-of-scope__details {
+  flex-grow: 1;
+  flex-shrink: 1;
+  font-weight: 500;
+}
+
+.group-list-item-out-of-scope__details--toggle {
```

Comment:
It feels to me like the `--toggle`, `--unavailable-message`, `--actions` and `--group-page-link` "modifiers" are really subcomponents rather than variants/modifiers, so they should be eg. `...__details-toggle`, `...-details-actions`.

The naming scheme we use is a variation of the classic [Block Element Modifier](http://getbem.com/introduction/) scheme. See that page for some more examples.

Reply:
Ok yeah, that makes sense.